### ISSUE TITLE
Cleanup is not accurate

### ISSUE THREAD

**Issue by tifish** [deepak-rathi/VS2017OfflineSetupUtility#15]
- Some old package should be keep.
- Some new package should be deleted.
- Some package should be deleted of all versions.

It seems cleanup by Catalog.json is OK. I write a simple batch file for it:

```batch
@echo off
setlocal EnableDelayedExpansion

set params=
for /d %%d in ("%~dp0Archive\*") do (
    if exist "%%d\Catalog.json" (
        set params=!params! --clean "%%d\Catalog.json"
    )
)

if not "!params!" == "" (
    "%~dp0vs_enterprise.exe" --layout "%~dp0" %params% --quiet --wait
)

endlocal
```

**Comment by deepak-rathi** [comment]
My laptop is broken, so i wont be able to test and fix issues for few months.

**Comment by deepak-rathi** [comment]
Fixed in release 1.4.2
https://github.com/deepak-rathi/VS2017OfflineSetupUtility/releases/tag/1.4.2.0
